src/memory/AmdFlash.cc

```diff
@@ -38,7 +38,7 @@ AmdFlash::AmdFlash(const std::string& name, Chip chip_,
 	: motherBoard(config.getMotherBoard())
 	, chip(chip_)
 	, sectorInfo(sectorInfo_)
-	, sz(sum(sectorInfo, &SectorInfo::size))
+	, sz(sum(chip.geometry.regions, [](Region r) { return r.count << r.size.exponent; }))
```

Comment:
This is a repeat of the above expression. Should we make this a member function of `Chip` instead?
(And then we can still cache the result of that expression here in `AmdFlash`).

Reply:
I will cache `size` on `Geometry`, together with `sectorCount`.